Project.toml

```diff
@@ -7,5 +7,8 @@ version = "0.1.0"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+NBodySimulator = "0e6f8da7-a7fc-5c8b-a220-74e902c310f9"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 ShallowWaters = "56019723-2d87-4a65-81ff-59d5d8913e3c"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
Ok, but eventually we should remove all these dependencies from the examples (well before JCon).

Reply:
yes!